Formula/fargatecli.rb

```diff
@@ -0,0 +1,25 @@
+class Fargatecli < Formula
+  desc "CLI for AWS Fargate"
+  homepage "https://github.com/awslabs/fargatecli"
+  url "https://github.com/awslabs/fargatecli/archive/v0.3.1.tar.gz"
+  sha256 "fd8627e6ab85d70f6b2633d8b35cfb1daad232ed0a29c72df57b9869810ab221"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+    prefix.install_metafiles
+  end
+
+  def caveats
```

Comment:
This should be in the upstream documentation

Reply:
i don't know if i am understand you mate, is your recommendation to removing the caveats from here?.  if so, is it a "must"?

Reply:
Yes https://docs.brew.sh/Formula-Cookbook#caveats